scripts/issue/217/claim_dropt.py

```diff
@@ -0,0 +1,35 @@
+from brownie import interface
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main():
+    """
+    claim dropt-3 from tree and redeem for bdigg -> digg
+    """
+
+    safe = GreatApeSafe(registry.eth.badger_wallets.treasury_ops_multisig)
+    safe.init_badger()
+
+    digg = registry.eth.treasury_tokens.DIGG
+    bdigg = interface.ISettV4h(registry.eth.treasury_tokens.bDIGG, owner=safe.address)
+    dropt = safe.contract(registry.eth.uma.DIGG_LongShortPair)
+    dropt_long = safe.contract(dropt.longToken())
+
+
+    safe.take_snapshot(tokens=[
```

Comment:
scripts looks clean, but will be great to add also in the snap `bcvxCRV`and `bveCVX`as they are claim also from the tree

Reply:
+1

Reply:
good idea 👍